processors/euctr_duplicates_remover/processor.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import logging
+logger = logging.getLogger(__name__)
+
+
+# Module API
+
+def process(conf, conn):
+    """Remove duplicate records from EUCTR"""
+
+    query = "SELECT eudract_number_with_country from euctr where eudract_number_with_country ~ '^EUCTR*';"
```

Comment:
So to make things clear, initially we had `EUCTR` as part of the ID, then we dropped it but also kept the old records instead of removing them. This led to a duplication of some EUCTR records, which is why we have to remove the old ones from the database.